### ISSUE TITLE
Not support field type datetime

### ISSUE THREAD

**Issue by measproem** [tarantool-php/mapper#32]
Any space field contain type `datetime` will problem

**Comment by nekufa** [comment]
hi @measproem, is this actual issue?\
can you provide an example of tarantool schema?\
never heard about datetime type in tarantool

**Comment by measproem** [comment]
> hi @measproem, is this actual issue? can you provide an example of tarantool schema? never heard about datetime type in tarantool

It is commonly issue for Tarantool client require implement[ MessagePack extension](https://www.tarantool.io/en/doc/latest/reference/internals/msgpack_extensions/) the data type of space column (UUID, DateTime, Decimal..)

**Comment by nekufa** [comment]
@measproem yep, looks like uuid, decimal and emails type are already implemented.\
https://github.com/tarantool-php/client/tree/master/examples/user_defined_type

i can't find an example of datetime type usage in tarantool, only lua module for datetime management

**Comment by rybakit** [comment]
@nekufa [Here](https://github.com/tarantool-php/client/tree/master/src/Packer/Extension) are all the extensions currently supported by the client. I recall starting work on adding datetime support but didn’t finish, as the Tarantool specification was still in development and subject to change at the time (I might still have some of that work). That said, implementing this and other missing types should be relatively straightforward. If anyone wants to give it a try, [this example](https://github.com/rybakit/msgpack.php/blob/master/src/Extension/TimestampExtension.php) of MessagePack's timestamp extension might serve as a good reference. There's also a [datetime extension example](https://github.com/rybakit/msgpack.php/blob/master/examples/datetime.php), though it's mostly for demo purposes and doesn’t align with the Tarantool specification. Finally, here's an [article](https://dev.to/tarantool/advanced-messagepack-capabilities-4735) on writing your own extension, which could also be useful.

**Comment by measproem** [comment]
> @measproem yep, looks like uuid, decimal and emails type are already implemented. https://github.com/tarantool-php/client/tree/master/examples/user_defined_type
> 
> i can't find an example of datetime type usage in tarantool, only lua module for datetime management

I forked other repo and solved but with [Rust ](https://github.com/measproem/alopecosa)and TypeScript just personal
